<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZamaxSO</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/Styles/icono.ico</normaloff>:/Styles/icono.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="pagesSW">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="setProcessPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="howManyProcessGB">
          <property name="maximumSize">
           <size>
            <width>636</width>
            <height>279</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="3" column="1">
            <widget class="QPushButton" name="numOfProcessPB">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="numOfProcessLB">
             <property name="text">
              <string>No. Procesos: </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="numOfProcessLE">
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="welcomeMsgLB">
             <property name="text">
              <string>Bienvenido al sistema de simulacion de procesamiento FCFS &quot;ZamaxSO&quot;. 
Por favor digite la cantidad de procesos que desea realizar.</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="logoLB">
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="recursos.qrc">:/Styles/Logo.PNG</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="executeProcessPage">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="2">
         <widget class="QGroupBox" name="finishedProcesGB">
          <property name="maximumSize">
           <size>
            <width>392</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Procesos Finalizados</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="2">
            <widget class="QLabel" name="resultColumnLB">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Resultado</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="operationColumnLB">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Operacion</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="idColumnLB">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>ID</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QScrollArea" name="finishProcessSA">
             <property name="maximumSize">
              <size>
               <width>392</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="widget">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>358</width>
                <height>443</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <layout class="QGridLayout" name="finishProcessGL"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="startPB">
          <property name="text">
           <string>Comenzar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="contOfBatchesLB">
          <property name="text">
           <string>Procesos Pendientes: </string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="globalContLB">
          <property name="text">
           <string>Contador: </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="readyProcessGB">
          <property name="maximumSize">
           <size>
            <width>392</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Listos</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1">
            <widget class="QLabel" name="timeColumnLB">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Tiempo</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="nameColumnLB">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>ID</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="ttLB_2">
             <property name="text">
              <string>TT</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QScrollArea" name="pendientProcessSA">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>358</width>
                <height>441</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="0">
                <layout class="QGridLayout" name="pendientProcessGL"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="executeProcessGB">
          <property name="maximumSize">
           <size>
            <width>392</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Procesos en ejecucion</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="3" column="0">
            <widget class="QLabel" name="ttLB">
             <property name="text">
              <string>Tiempo Transcurrido: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="opLB">
             <property name="text">
              <string>Operacion:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="tmLB">
             <property name="text">
              <string>Tempo Maximo:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="nameLB">
             <property name="text">
              <string>ID: </string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="trLB">
             <property name="text">
              <string>Tiempo Restante: </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="blockedLB">
          <property name="text">
           <string>Cola de Bloqueados</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="finishPB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Terminar</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="QScrollArea" name="blockedSA">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1162</width>
             <height>98</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="1" column="0">
             <layout class="QGridLayout" name="blockedGL"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="showResultsPage">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QScrollArea" name="resultsSA">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1162</width>
             <height>716</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <layout class="QGridLayout" name="resultsGL"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="bcpPage">
       <layout class="QGridLayout" name="gridLayout_24">
        <item row="1" column="0">
         <widget class="QTableWidget" name="bcpTW">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1100</width>
            <height>650</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 200, 11);
font: 14pt &quot;Century Gothic&quot;;</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::AllEditTriggers</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="bcpTitleLB">
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 200, 11);
font: 14pt &quot;Century Gothic&quot;;</string>
          </property>
          <property name="text">
           <string>Tabla de Bloque de Control de Procesos</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
